client/state/activity-log/utils.js

```diff
@@ -60,3 +65,14 @@ export const queryToFilterState = query => {
 		page && page > 0 && { page }
 	);
 };
+
+export const filterStateToTokens = filter => {
+	const apiQuery = filterStateToApiQuery( filter ),
+		tokens = [];
+	toPairs( omit( apiQuery, 'number' ) ).forEach( pair => {
```

Comment:
if this slipped through the normal commit process with poor formatting it may be a bug in my #24906

Reply:
it did slip through - i was not alerted about poor formatting here.

Reply:
hm. I probably messed up. thanks